package.json

```diff
@@ -76,6 +76,7 @@
     "@commitlint/cli": "6.1.3",
     "@commitlint/config-conventional": "6.1.3",
     "@dcos/eslint-config": "0.3.0",
+    "@dcos/ui-kit": "0.2.0",
```

Comment:
Whenever https://github.com/dcos-labs/ui-kit/pull/21 is merged a new version of UI Kit needs to be released and then here I will need to bump to the newly released version.

Reply:
dcos-labs/ui-kit#66 is the update on this.